### PR TITLE
Added setting for top menu customization

### PR DESCRIPTION
**Pull request by stevegaarder**
Here is the code for the custom menu setting.  I note that lint complains about a line that was already in community.rb

### CODE REVIEW

File: app/models/community.rb [hunk 1]
```diff
@@ -33,6 +33,7 @@ class Community < ApplicationRecord
     {key: :time_zone, type: :time_zone, required: true, default: "UTC"},
     {key: :default_landing_page, type: :enum, options: %w[meals directory reservations wiki],
      default: "directory", required: true},
+    {key: :top_menu_customizations, type: :text},
```
Comment (smoyth) [inline app/models/community.rb]: For consistency with the rest of the code I think this should be called `main_nav_customizations`.

File: config/locales/en/custom_fields.yml [hunk 2]
```diff
@@ -49,6 +49,7 @@ en:
     hints:
       community:
         settings:
+          top_menu_customizations: "Enter links in Markdown format. Example: [Link Name](https://example.com). If the link name matches an existing one it will override the default URL. A URL of 'none' will remove a link completely. All other text and markup will be ignored."
```
Comment (smoyth) [inline config/locales/en/custom_fields.yml]: Let's add this to the front to be a bit more explicit. "Use this setting to edit the links in the top menu."

Comment (smoyth) [reply config/locales/en/custom_fields.yml]: There should also be a translation entry above under `labels` for the field label. Let's use `Top Menu Customizations`. While we use `nav` in the code, "Menu" will be more recognizable to users.

**Comment by smoyth** [comment]
Changes look good. Can you please paste in a screenshot of the updated part of the settings screen? I'll take a look and then I can merge this.

**Comment by stevegaarder** [comment]
Screenshot_custom_menu_setting_2019_10_20

**Comment by smoyth** [comment]
Gorgeous! Merging!

**Pull request merged**